app/dashboard/management/commands/cleanup_dupe_profiles.py

```diff
@@ -0,0 +1,61 @@
+'''
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from dashboard.models import Profile
+
+
+def combine_profiles(p1, p2):
+    # p2 is the delete profile, p1 is the save profile
+    # switch if p2 has the user
+    if p2.user:
+        tmp = p2
+        p2 = p1
+        p1 = tmp
```

Comment:
We can do `p1, p2 = p2, p1` in python to swap.

Reply:
thanks!1 15afb6c9d7f5edd15a6c1444d9529ebdc49a2ba8